addons/account/migrations/9.0.1.1/post-migration.py

```diff
@@ -417,6 +417,10 @@ def migrate(env, version):
     WHERE child_depend IS True
     """)
 
+    # In v9, percentages are expressed as percentage
```

Comment:
Better to say: `In v9, percentages are expressed as hundred-based percentage, not one-based percentage`